h11/_headers.py

```diff
@@ -1,5 +1,5 @@
 import re
-from typing import AnyStr, cast, List, overload, Sequence, Tuple, TYPE_CHECKING, Union
+from typing import List, overload, Sequence, Tuple, TYPE_CHECKING, TypeVar, Union
```

Comment:
My IDE was telling me `AnyStr` and `cast` were not used, I used the opportunity of changing the line to remove them